CHANGELOG.md

```diff
@@ -27,7 +27,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
-Nil.
+- Add support for `Region#associated_continent` to return the containing continent of the region
```

Comment:
Could you update the changelog entry to link this PR?
```markdown
- Add support for `Region#associated_continent` to return the containing continent of the region [#43](https://github.com/Shopify/worldwide/pull/43) by @Flufd
```
Attribution is entirely up to you.